sdk/cognitivelanguage/test-resources.json

```diff
@@ -24,41 +24,41 @@
         // Developers should define the following environment variables to run tests locally.
```

Comment:
just want to check - are the changes in this file intentional?

Reply:
Yes. Our test framework tries a number of environment variables beginning with a service directory-prefixed name like `COGNITIVELANUAGE_{name}` then just `{name}` then `AZURE_{name}` for backward compatibility with our older system. Because of that, and so I could easily share _test-resources.json_ with (or from) you folks, I put this back to creating (eventually, when we have dynamic deployment) `AZURE_{name}` variables.

/cc @mshaban-msft